src/event-bridge/event-bus.ts

```diff
@@ -213,6 +225,22 @@ abstract class EventBusBase<E extends EventBusRuleInput>
     };
   }
 
+  eventBus: EventBusTargetIntegration<
+    E,
+    aws_events_targets.EventBusProps | undefined
+  > = {
+    target: (props, targetInput?) => {
+      if (targetInput) {
+        throw new Error("Event bus rule target does not support target input.");
+      }
```

Comment:
Not sure what this means? Why is it available as an argument then?

Reply:
`PIpe` takes any integration and event bus is an integration. however, event bus cannot accept transformed input (`.map`).

I updated the types to fail when an event bus is given. Keeping the error because someone can ignore the type error.